internal/server/routes.go

```diff
@@ -106,24 +106,16 @@ func (s *Server) GenerateRoutes(promRegistry prometheus.Registerer) Routes {
 	get(a, noAuthnNoOrg, "/api/version", a.Version)
 	get(a, noAuthnNoOrg, "/api/server-configuration", a.GetServerConfiguration)
 
-	// while an org is required for these endpoints, they return fake data when the org is not supplied
-	noAuthnOptOrg := apiGroup.Group("/",
```

Comment:
I combined this with `noAuthnWithOrg` with org for now. I had made that change before I brought in these changes from `main` to fix a panic in one of the test cases.

Reply:
there's no panic anymore. I do prefer to leave them separate as it's clear what endpoints need to be updated later.

Reply:
Are you sure the current grouping is correct? It seems strange that get provider would be in one group while list providers is in a different one. Also the jwk route should probably be in the same group.

If anyone adds routes between now and when we add fake data they likely won't have any idea which group it goes into, and could put it in the wrong group.

It seems like either way we'll have to change these groups later, so why split them now?